src/index.ts

```diff
@@ -98,98 +98,13 @@ function interceptor(ctx) {
     FakeXMLHttpRequest.prototype.send.apply(this, arguments);
 
     if (ctx.pretender.checkPassthrough(this)) {
-      var xhr = createPassthrough(this);
+      let xhr = createPassthrough(this, ctx.pretender._nativeXMLHttpRequest);
       xhr.send.apply(xhr, arguments);
     } else {
       ctx.pretender.handleRequest(this);
     }
   };
 
-
-  function createPassthrough(fakeXHR) {
-    // event types to handle on the xhr
-    var evts = ['error', 'timeout', 'abort', 'readystatechange'];
-
-    // event types to handle on the xhr.upload
-    var uploadEvents = [];
-
-    // properties to copy from the native xhr to fake xhr
-    var lifecycleProps = ['readyState', 'responseText', 'responseXML', 'status', 'statusText'];
-
-    var xhr = fakeXHR._passthroughRequest = new ctx.pretender._nativeXMLHttpRequest();
```

Comment:
`ctx.pretender._nativeXMLHttpRequest` was a closure and now it is an argument for the function.